packages/@webex/plugin-meetings/src/roap/index.ts

```diff
@@ -231,14 +231,16 @@ export default class Roap extends StatelessWebexPlugin {
               headers,
             } = remoteSdp.roapMessage;
 
-            roapAnswer = {
-              seq: answerSeq,
-              messageType,
-              sdp: sdps[0],
-              errorType,
-              errorCause,
-              headers,
-            };
+            if (messageType === ROAP.ROAP_TYPES.ANSWER) {
```

Comment:
Again, another thing I've found while testing this area - we might get something else than an ANSWER (for example ERROR) and then the `sdps` array will be empty so we would crash. I've added a unit test for this too.